recipe/meta.yaml

```
@@ -24,9 +24,9 @@ requirements:
     - cmake
     - ninja
   host:
-    - eigen
+    - {{ backend }}
```

Just to make the linter happy
```suggestion
    - {{ backend|default("casadi") }}
```